fresnel_plane/materials/scripts/fresnel_vp.glsl

```diff
@@ -0,0 +1,37 @@
+#version 150
+
+in vec4 uv0;
+in vec4 position;
+in vec3 normal;
+
+uniform mat4 worldViewProjMatrix;
```

Comment:
In the current wave_gazebo, it has predefined variables like gl_ModelViewProjectionMatrix. This may just be because this is done in #version 150. Same with using `in` and `out` instead of `varying`.

Reply:
yep I'm doing all in version 150 for now to get something working first. We can backport to 120 if needed.